partialWaveFit/fitResult.cc

```diff
@@ -574,9 +572,6 @@ fitResult::intensityErr(const string& waveNamePattern) const
 	if (not _covMatrixValid or (nmbAmps == 0)) {
 		return 0.;
 	}
-	if(nmbAmps == 1) {
-		return intensityErr(prodAmpIndices[0]);
-	}
```

Comment:
Here some fixing is indeed required, but I am not sure that removing those lines (and thus reverting a4e9e17cbd055ec9a0f2793bdac5dcf083626da4) is a good choice. Initially commit a4e9e17cbd055ec9a0f2793bdac5dcf083626da4 was made to allow plotting intensities (and all other information of a single spin-density matrix element) with errors also for fit results without integral matrices. This functionality needs to be kept.